submissions.html

```diff
@@ -0,0 +1,190 @@
+<p class="font-medium">Palermo, Palazzo Steri, University of Palermo, Italy<br>October 28-29, 2025<br></p>
+<p class="font-medium">Call for Submissions</p>
+
+We call for submissions of original productions in two separate routes:
+<ol>
+  <li>Scientific Papers: We welcome scientific papers that address topics related to quantum computing and musical creativity. We plan to publish a conference proceedings with accepted papers (more information soon). Your paper, if accepted, will be presented in an oral format during the conference. Short musical demonstrations can be part of the presentation, in case that contributes to the exposition of your scientific research outcomes.<br><br>
+
+  We welcome papers analysing/reporting on scientific, technical, artistic and/or practice-based developments in <strong>Quantum Computer Music</strong>. Topics may include, but are not limited to:
+
+  <ul>
+    <li>Quantum Algorithms in Music</li>
+    <li>Quantum-computing Aided Composition</li>
+    <li>Quantum Representations of Audio</li>
+    <li>Quantum Signal Processing</li>
+    <li>Musical Instruments</li>
+    <li>Quantum Computing Software for Music</li>
+    <li>Sonification of Quantum Processes and Phenomena</li>
+    <li>Live performance with Quantum Computing</li>
+    <li>Music Information Retrieval</li>
+    <li>Hybrid Quantum-Classical Architectures</li>
+    <li>Music Theory and Quantum Computing Intersections</li>
+    <li>Quantum Aesthetics</li>
+    <li>Brain Quantum Computer Interfaces</li>
+    <li>Quantum Machine Learning</li>
+    <li>Quantum Natural Language Processing</li>
+    <li>Music Industry Applications</li>
+    <li>Quantum Computational Musicology</li>
+    <li>Quantum Computer Music in Education</li>
+  </ul>
+      </li>
+ 
+      <br>
+      All paper submissions will be peer-reviewed (double blind) for originality, academic standards, appropriateness of topic, readability (language), and organisation.<br><br>
+
+      <li>Musical Works: We also welcome short musical compositions (fixed-media) and/or performances and/or demonstrations that showcase the intersection of quantum computing and musical creativity. For this route, a detailed <strong>manuscript</strong> explaining the idea must be submitted. 
+
+        <br><br>Please follow the instructions under the <strong>Manuscript Instructions</strong> section below in this page, for the expected format and content of the manuscript.
+
+        If the work is accepted, it will be diffused/performed during the conference, at our conference auditorium - <a href='https://www.unipa.it/servizi/cerimoniale/concessione-spazi-steri/Spazi_Steri/'><u>Salla Delle Capriate</u></a><br><br>
+
+     All manuscripts will be reviewed for originality, academic standards, appropriateness of topic, readability (language), and technical feasibility. The audio/video file will be assessed for its artistic/conceptual quality and relevance to the manuscript and the topic.<br><br> 
```

Comment:
@Itaborala @medusamedusa 
I still think this may be a bit too much for the music submissions: to say that they will be reviewed for academic standards. That is, I am afraid it will make ppl give up on submitting a music proposal. But it is easy to defend if you say that these manuscripts would also appear on the proceedings (including the potentially publisher proceedings version, though not sure how yet).

Reply:
Thats an interesting point @omarcostahamido. I guess the idea is that the manuscript should really clarify the connection between the artistic concept/process and quantum computing, either technical, or aesthetic. To me, this is an academic standard - but perhaps a different one from the paper submission, more practice-based. Not sure about the proceedings.

If you feel it may scare people away, I'd vote to remove it.

Reply:
did you receive the email that i just _bumped_ to quantummusicsubmission?